Makefile

```diff
@@ -9,7 +9,10 @@ swagger-ui: ## docs :
 	./scripts/swagger-ui.sh
 
 cargo-clippy: ## lint :
-	cargo clippy -- --no-deps -D warnings
+	cargo clippy -- \
+	    --no-deps \
+	    --allow clippy::derive_partial_eq_without_eq \
```

Comment:
no reason to accept this lint.
waiting for derive_partial_eq_without_eq to be removed from the default rule.

rf. https://github.com/rust-lang/rust-clippy/issues/9063